src/main/java/com/solvd/blog/tx/TxUsers.java

```diff
@@ -46,4 +46,10 @@ public User update(final User user) {
         return this.users.update(user);
     }
 
+    @Override
+    @Transactional(readOnly = true)
+    public Boolean isExists(final String email) {
```

Comment:
@hizmailovich `is` prefix can be suppressed here, just `exists` will be more elegant